packages/volto/src/components/theme/Icon/Icon.jsx

```diff
@@ -56,8 +56,9 @@ const Icon = ({
     onClick={onClick}
     id={id}
     aria-hidden={ariaHidden}
+    aria-label={title || ''}
     dangerouslySetInnerHTML={{
-      __html: title ? `<title>${title}</title>${name.content}` : name.content,
+      __html: title ? `<title >${title}</title>${name.content}` : name.content,
```

Comment:
```suggestion
      __html: title ? `<title>${title}</title>${name.content}` : name.content,
```
Seems like a useless change to me, is it not?